bedrock/firefox/views.py

```diff
@@ -226,10 +227,19 @@ def all_downloads(request, channel):
 
 
 def firefox_os_index(request):
+
+    locale = l10n_utils.get_locale(request)
+    lang_file = 'firefox/os/index-new'
+    old_home = 'firefox/os/index.html'
+    new_home = 'firefox/os/index-new.html'
+
     if waffle.switch_is_active('firefox-os-index-2015'):
```

Comment:
Nope. Switch name is just a string and can be anything. We just need to be consistent about it, especially if a single switch will turn on things in multiple areas of the code.